### ISSUE TITLE
Can you replace FCM message push

### ISSUE THREAD

**Issue by yangjl114** [element-hq/synapse#17100]
**Description:**

Because FCM push is limited, now want to know how to connect to another service, such as engagelab

**Comment by dklimpel** [comment]
IMO this is a feature request to the push gateway [Sygnal](https://github.com/matrix-org/sygnal).

**Comment by yangjl114** [comment]
@dklimpel   Yes, I found sygnal, but I didn't find how to configure sygnal's mailing address in matrix-synapse homeserver.yaml. Do you know that? I really hope to get your help

**Comment by dklimpel** [comment]
The description how it works is there: https://spec.matrix.org/v1.10/push-gateway-api/

There is no confirmation in Synapse for that. The client gives the information (and URL) of the pusher to Synapse.

**Comment by yangjl114** [comment]
@dklimpel   Hello, now sygnal can listen to matrix-synapse messages and push them to the app side, but the message received by the app side is empty, without title or specific message content. May I ask why this is? I really hope to get your help, thank you very much

**Comment by dklimpel** [comment]
Please have a look at: https://element-hq.github.io/synapse/latest/usage/configuration/config_documentation.html#push

You can set:

```yaml
push:
  enabled: true
  include_content: true
```
